.circleci/cimodel/data/simple/docker_definitions.py

```diff
@@ -26,6 +26,7 @@
     "pytorch-linux-bionic-rocm4.0.1-py3.6",
     "pytorch-linux-bionic-rocm4.1-py3.6",
     "pytorch-linux-bionic-rocm4.2-py3.6",
+    "pytorch-linux-bionic-rocm4.3.1-py3.6",
```

Comment:
Is there any reason not to just use the already existing `pytorch-linux-bionic-rocm4.3-py3.6` tag instead of having to denote by the patch version?

Reply:
@jeffdaily @sunway513 Any strong reason to not break from earlier convention and simply use the rocm4.3 tag even if the ROCm version is 4.3.1?

Reply:
I'd suggest to keep the rocm minor version in the naming.

Reply:
@seemethere Then can we request you to please create a ROCm4.3.1 tag? Thanks!

Reply:
> I'd suggest to keep the rocm minor version in the naming.

Any particular reason why?

> @seemethere Then can we request you to please create a ROCm4.3.1 tag? Thanks!

I’ve created it.

Reply:
@seemethere , thanks for the help. We'd like to keep a more accurate version mapping, for maintenance consideration.